portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,298 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="description" content="" />
```

Comment:
This meta is used by search engine to show brief summary of your website (for SEO purpose), however as this is just a sample project it is also fine if you don't include it